### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by cqc-melf**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,85 @@
...
+
+`pytket-aqt` is available for Python 3.8, 3.9 and 3.10, on Linux, MacOS
+and Windows. To install, run:
+
+```pip install pytket-aqt```
+
+This will install `pytket` if it isn't already installed, and add new classes
+and methods into the `pytket.extensions` namespace.
+
+The [example notebooks](https://github.com/CQCL/pytket-aqt/tree/develop/examples) also demonstrate some of the key features of pytket-aqt.
```
Comment (cqc-alec) [inline README.md:26]: These don't exist do they?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,85 @@
...
+`mypy` locally on any changed files before submitting a PR. Because of the way
+extension modules embed themselves into the `pytket` namespace this is a little
+complicated, but it should be sufficient to run the script `modules/mypy-check`
+(passing as a single argument the root directory of the module to test). The
+script requires `mypy` 0.800 or above.
+
+#### Linting
+
+We use [pylint](https://pypi.org/project/pylint/) on the CI to check compliance
+with a set of style requirements (listed in `modules/.pylintrc`). You should run
```
Comment (cqc-alec) [inline README.md:71]:
```suggestion
with a set of style requirements (listed in `.pylintrc`). You should run
```

File: docs/changelog.rst [hunk 3]
```diff
@@ -0,0 +1,94 @@
+Changelog
+~~~~~~~~~
+
+0.24.0 (September 2022)
```
Comment (cqc-alec) [inline docs/changelog.rst:4]: October?

File: docs/changelog.rst [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+-----------------------
+
+* Updated pytket version requirement to 1.7.
```
Comment (cqc-alec) [inline docs/changelog.rst:7]: The change in `setup.py` is to 1.6; are we going to wait for 1.7 and update there?

File: pytest.ini [hunk 5]
```diff
@@ -0,0 +1,3 @@
+[pytest]
+filterwarnings =
+    ignore:Call to deprecated create function FieldDescriptor
```
Comment (cqc-alec) [inline pytest.ini:3]: Is this needed?

File: .github/workflows/build_and_test.yml [hunk 6]
```diff
@@ -0,0 +1,142 @@
...
+        ./.github/workflows/build-test nomypy
+    - name: Set up Python 3.10
+      if: github.event_name == 'push' || github.event_name == 'pull_request'
+      uses: actions/setup-python@v3
+      with:
+        python-version: '3.10'
+    - name: Build and test (3.10)
+      if: github.event_name == 'push' || github.event_name == 'pull_request'
+      run: |
+        ./.github/workflows/build-test nomypy
```
Comment (cqc-alec) [inline .github/workflows/build_and_test.yml:59]: Tests are not currently running on Windows. These calls should have `shell: bash` in the workflow to fix this I think.

File: docs/changelog.rst (hunk 4 above)
Comment (cqc-melf) [reply docs/changelog.rst:7]: Yes

File: pytest.ini (hunk 5 above)
Comment (cqc-melf) [reply pytest.ini:3]: Will check if we need it, good spot

File: .github/workflows/build_and_test.yml (hunk 6 above)
Comment (cqc-melf) [reply .github/workflows/build_and_test.yml:59]: Thank you, will try that!

File: README.md (hunk 1 above)
Comment (cqc-melf) [reply README.md:26]: no, we don't have them. Thank you

**Pull request merged**